pages/examples/withplatform/processGraphic.html

```diff
@@ -52,7 +52,8 @@
                 }
               },
               { },
-              { positionX: 85,
+              { image: '../images/vat.png',
```

Comment:
this shouldnt be necessary though, right?

Reply:
yeah i took it off